### DISCUSSION TITLE
Failed to load some of the config files

### DISCUSSION THREAD

**Discussion by jagannathamkrishna** [egovernments/DIGIT-DevOps#1705] [Q&A] [answered]
Hi i am facing an issue in deploying quicksetup VM while deploying the pods egov-persister, egov-indexer are in crashbackloop because of Failed to load some of the config files. 

![error](https://github.com/egovernments/DIGIT-DevOps/assets/133766435/b04c58fb-0d49-4906-be78-edb05db04cc6)

Please help in this issue

**Comment by talele08** [comment, marked as answer]
The files should be present in the config repo which is cloned using git-sync container. The config repo is overwritten in env.yaml file. Please refer this `[link](https://github.com/egovernments/DIGIT-DevOps/blob/e666d18f29c6021c809eb7af5f16f075a2ac0ab6/deploy-as-code/helm/environments/dev.yaml#L258)`
